rdkit/Chem/PandasTools.py

```diff
@@ -350,7 +353,11 @@ def LoadSDF(filename, idName='ID', molColName='ROMol', includeFingerprints=False
     if mol.HasProp('_Name'):
       row[idName] = mol.GetProp('_Name')
     if smilesName is not None:
-      row[smilesName] = Chem.MolToSmiles(mol, isomericSmiles=isomericSmiles)
+      try:
+        row[smilesName] = Chem.MolToSmiles(mol, isomericSmiles=isomericSmiles)
+      except:
+        print("No valid smiles could be generated for molecule " + str(i))
+        continue
```

Comment:
The consequence of this is that no row is added to the output. I think it would be better to either add an empty string or None to the row.

Reply:
Thank you for the suggestion. I have changed the code such that a None is added